src/variables/index.ts

```diff
@@ -45,10 +83,37 @@ export class Variables extends Panel {
    */
   private _resizeBody(msg: Widget.ResizeMessage) {
     const height = msg.height - this._header.node.offsetHeight;
-    this._body.node.style.height = `${height}px`;
+    this._table.node.style.height = `${height}px`;
+    this._tree.node.style.height = `${height}px`;
   }
 }
 
+/**
+ * Convert a variable to a primitive type.
+ * @param variable The variable.
+ */
+export const convertType = (variable: VariablesModel.IVariable) => {
+  const { type, value } = variable;
+  switch (type) {
+    case 'int':
+      return parseInt(value, 10);
+    case 'float':
+      return parseFloat(value);
+    case 'bool':
+      return value;
+    case 'str':
+      return value.slice(1, value.length - 1);
+    case 'list':
+      return type;
+    case 'dict':
+      return type;
+    case 'function':
+      return type;
+    default:
+      return value;
```

Comment:
Why make `list`, `dict` and `function` special, and not keep them in the `default`?

```typescript
default:
  return type;
```

There could be plenty of other types.